ros_buildfarm/templates/snippet/rosdep_init.Dockerfile.em

```diff
@@ -0,0 +1,8 @@
+RUN mkdir -p /etc/ros/rosdep/sources.list.d
+
+@[for url in custom_rosdep_urls]@
+RUN (cd /etc/ros/rosdep/sources.list.d/ && wget --progress=dot @url )
+@[end for]@
+@[if not custom_rosdep_urls]@
+RUN rosdep init
```

Comment:
What does `rosdep init` do other than create the /etc/ros/rosdep/sources.list.d/20-default file from the default one on ros/rosdistro? It didn't seem to me like it did anything else, which is why I did it that way.
It'd be easy enough to change it to run `rosdep init` first and then overwrite 20-default.list with the user-specified one if desired though.